chapters/request_size.rst

```diff
@@ -0,0 +1,133 @@
+=================
+Request Size
+=================
+
+``request_size`` is the parameter that we could set for the Flow.
+It defines the number of data sent into parallel ``Peas``.
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Request Size
+------------
+Jina defines the ``request_size`` as the parameter at client side. By setting the ``request_size`` in Flow's API
+we could define the number of data sent to each ``Peas``.
+
+Batch Size
+------------
+Batch size, commonly used in machine learning, often refers to the number of data that we feed into
+the model in one iteration.
+
+In Jina, ``batch_size`` is used at ``Driver`` and ``Executor`` level. Where
+we align the usage of machine learning terms for ``batch_size``.
+
+Before you start
+------------------
+Make sure you intall Numpy and latest version of Jina on your local machine.
```

Comment:
Latest version of Jina already installs numpy as it is a core dependency

Reply:
Sure, will fix